### PR TITLE
Update ServiceBusSessionManager.java

### PR DESCRIPTION
**Pull request by lgw-spyrosoft**
bug description:
#33867
https://github.com/Azure/azure-sdk-for-java/issues/33867

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionManager.java [hunk 1]
```diff
@@ -235,8 +235,11 @@ public void close() {
         final List<Mono<Void>> closeables = sessionReceivers.values().stream()
             .map(receiver -> receiver.closeAsync())
             .collect(Collectors.toList());
-
-        Mono.when(closeables).block(operationTimeout);
+        
+        if(!closeables.isEmpty()){
+            Mono.when(closeables).block(operationTimeout);
+        }
+
```
Comment (lgw-spyrosoft) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionManager.java:242]: its a mitigation, a little improvement, but it doesnt solve the main issue - "calling blocking methods within nonblocking context"

**Comment by KnyGoo** [comment]
Any updates? We have major issues using session manager because consumer stops working at random times. At this moment we have to restart our application to start the scheduler again.

**Comment by lgw-spyrosoft** [comment]
@KnyGoo try to update dependency - it seems like its solved 
"com.azure:azure-messaging-servicebus:7.13.2"

**Comment by KnyGoo** [comment]
Hi @lgw-spyrosoft ,

I just updated to 7.13.3 and the situation is worse than before...
- At this moment with 7.12 we receive messages but after soms time it stops
- With 7.13.3 it stops almost immediately after server startup...

With kind regards,

Kenny

**Pull request closed without merging**